README.md

```diff
@@ -165,5 +196,11 @@ relationships between the two are set up in Active Record.
 
 Write your code here:
 ```ruby
-# code here
+1. lamp = Lamp.create(wishes_remaining: 3), genie = Genie.create(name: "Genie")
+2. genie.lamp = lamp
+3. lamp.update(wishes_remaining: 1)
+4. jafar = Genie.create(name: "Jafar")
+jafar.lamp.create(wishes_remaining: 3)
+5. genie.lamp.delete
```

Comment:
This is a contrived example, and you most likely won't need to do something like this again, but I think what the question is looking for is something like:
```rb
genie.update(lamp: nil)
``